pkg/clusteragent/orchestrator/controller.go

```diff
@@ -152,6 +158,11 @@ func (o *Controller) Run(stopCh <-chan struct{}) {
 	log.Infof("Starting orchestrator controller")
 	defer log.Infof("Stopping orchestrator controller")
 
+	if err := o.runLeaderElection(); err != nil {
+		log.Errorf("error trying run or check the leader engine and status: %s", err)
```

Comment:
nit: logs should be capitalized

```suggestion
		log.Errorf("Error running the leader engine: %s", err)
```